pkg/plugins/resources/shell/environments.go

```diff
@@ -0,0 +1,70 @@
+package shell
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/sirupsen/logrus"
+)
+
+type Environments []Environment
+
+func (e *Environments) ToStringArray() []string {
```

Comment:
```suggestion
func (e Environments) ToStringArray() []string {
```

Slices are already references, you don't need to use a pointer to manipulate them without copy

Reply:
Good point, thanks for the advice, I'll try to remember next time